posts/2024/<your-path>.md

```diff
@@ -0,0 +1,36 @@
+---
+media:
+ - mastodon-eu-freiburg
+ - matrix-eu-announce
+ - bluesky-galaxyproject
+ - linkedin-galaxyproject
+
+mentions:
+ mastodon-eu-freiburg:
+  - galaxyproject@mstdn.science
+ bluesky-galaxyproject:
+  - galaxyproject.bsky.social
+
+hashtags:
+ mastodon-eu-freiburg:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - EOSC
+  - EuroScienceGateway
+ bluesky-galaxyproject:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - EOSC
+  - EuroScienceGateway
+ linkedin-galaxyproject:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - EOSC
+  - EuroScienceGateway
+---
+Galaxy Project is launching a new initiative called BRC-Analytics (https://brc-analytics.org) funded by NIAID. It is a platform for the analysis of pathogen, host, and vector data. Initially it will serve organisms that were previously included in VEuPathDb resource. BRC-Analytics is a collaboration between the Galaxy Project, UCSC Genome Browser, and the Texas Advanced Computing Center (TACC) with active participation from NCBI Datasets and Comparative Genomics Resource. We are just starting - so please be patient with us we are rapidly developing tools for data exploration and analysis.
```

Comment:
```suggestion
Galaxy Project is launching a new initiative called BRC-Analytics (https://brc-analytics.org) funded by NIAID. It is a platform for analysing pathogen, host, and vector data. Initially, it will serve organisms that were previously included in VEuPathDb resource. BRC-Analytics is a collaboration between the Galaxy Project, UCSC Genome Browser, and the Texas Advanced Computing Center (TACC) with active participation from NCBI Datasets and Comparative Genomics Resource. 

We are just starting - so please be patient with us we are rapidly developing tools for data exploration and analysis.
```

If you look at the preview you see a split in mastodon that is not optimal, this will force a thread before "we are just"

Reply:
It really should be "for the analysis" but, ok